app/screens/create_channel/create_channel.js

```diff
@@ -60,7 +61,7 @@ export default class CreateChannel extends PureComponent {
 
         this.rightButton.title = context.intl.formatMessage({id: 'mobile.create_channel', defaultMessage: 'Create'});
 
-        if (props.channelType === General.PRIVATE_CHANNEL) {
+        if (props.closeButton && props.isModal) {
```

Comment:
Commented re: `isModal` missing above, but this logic seems to be getting more confusing. If a `closeButton` is provided, is there a reason we can't just render it unconditionally?